sycl/test-e2e/ESIMD/api/svm_gather_scatter.cpp

```diff
@@ -123,6 +123,7 @@ int main(void) {
   Pass &= test<int32_t, 16>(Q);
   Pass &= test<int32_t, 32>(Q);
 #endif
+
```

Comment:
Huge nitpick: Remove whitespace changes in this file

Reply:
Ok, removed in f541a69d936f5c31befd154497f8f1e9f1af7b57